Sources/_CryptoExtras/RSA/RSA+BlindSigning.swift

```diff
@@ -113,6 +113,16 @@ extension _RSA.BlindSigning {
             self.backing = backing
             self.parameters = parameters
         }
+
+        public struct Primitives {
+            public var n: Data
+            public var e: Data
```

Comment:
For this type, can we use the long names for these fields? `modulus` and `public exponent`. Let's also make this type `Hashable`, `Sendable`, and give it a `public init`. Same for all the others.

Reply:
I used `n` and `e` to be consistent with the `init(n:, e:)` initialiser but happy to change it

Reply:
Yeah, I regret having done that that way, but too late now!